blogs/2022-06-28-tips-kafka-streams-distributed.md

```diff
@@ -104,9 +110,9 @@ Now, let's handle the data stored in Kafka. The main logic is a [Kafka Stream](h
 - If there is a new `WorkerInstance`, we look at the partition assigned to this one. If there is an overlap between this instance's partitions and the previous one, we know that the previous `WorkerInstance` is dead. In Kafka, you can't have two consumers on the same partition.
 - We only need to look at the affected tasks in this `WorkerInstance` and resend them for processing.
 
-Et voila! We have detection of dead consumers using just the Kafka API. 🎉 
+Et voilà! We have detection of dead consumers using just the Kafka API. 🎉 
 
-## State store `all()`
+## Beware of State Store `all()`
 
 We use a [GlobalKTable](https://kafka.apache.org/31/documentation/streams/developer-guide/dsl-api.html#streams_concepts_globalktable) to detect [flow trigger](/docs/developer-guide/triggers/flow.html). For all the flows on the cluster, we test all the flow's [conditions](/docs/developer-guide/conditions/) to find matching flows. For this, we are using an API to fetch all flows from a `GlobalKTable` using `store.all()` that returns all the flows, fetching RocksDB.
```

Comment:
What does this mean "that returns all the flows, fetching RocksDB."? Does that mean to say fetching from RocksDB?

Reply:
It means that you `store.all()` will fetch all the flows from RockDb (internal database from kafka stream), we try to highlight after that fetching from rockdb have an issue (deserialization), that don't seems to be hard at first